src/test/java/org/roaringbitmap/buffer/TestIterators.java

```diff
@@ -245,5 +250,31 @@ public void testEmptySkips() {
     PeekableIntIterator it = bitmap.getIntIterator();
     it.advanceIfNeeded(0);
   }
+  
+  @Test
+  public void testIteratorsOnLargeBitmap() throws IOException {
+      MutableRoaringBitmap bitmap = new MutableRoaringBitmap();
+
+      int inc = Short.MAX_VALUE;
+
+      for (long i = -Integer.MIN_VALUE; i < Integer.MAX_VALUE; i += inc) {
```

Comment:
@gpunti Fix and test are correct but there is small anomaly hidden with overflow of  `- Integer.MIN_VALUE`, see http://stackoverflow.com/questions/12535095/java-integers-min-value-negative-then-compare